webclient/index.html

```diff
@@ -64,7 +70,7 @@
       // Listener for Send button
       $('form').submit(function(){
         // Selector for the text input box
-        var inputBoxSel = '#m';
+        var inputBoxSel = '#message';
```

Comment:
Good, thanks for cleaning up this ID. It is a remnant of the tutorial that I started with.